apps/taxes/src/modules/avatax/order-confirmed/avatax-order-confirmed-payload-lines-transformer.ts

```diff
@@ -31,18 +31,11 @@ export class AvataxOrderConfirmedPayloadLinesTransformer {
     });
 
     if (order.shippingPrice.net.amount !== 0) {
-      // * In AvaTax, shipping is a regular line
-      const shippingLine: LineItemModel = {
+      const shippingLine = avataxShippingLine.create({
         amount: order.shippingPrice.gross.amount,
-        taxIncluded: true,
-        itemCode: SHIPPING_ITEM_CODE,
-        /**
-         * * Different shipping methods can have different tax codes.
-         * https://developer.avalara.com/ecommerce-integration-guide/sales-tax-badge/designing/non-standard-items/\
-         */
         taxCode: config.shippingTaxCode,
-        quantity: 1,
-      };
+        taxIncluded: true,
```

Comment:
question: Is this a good assumption in general? Sometimes you need to calculate different VAT for shipping (e.g. you're selling books with lower VAT rate and shipping is included a as a separate line with higher tax rate)

Reply:
Yes, but this is the `OrderConfirmed` event, when already all the taxes were calculated. We just need to pass the values to AvaTax. Otherwise, an extra tax gets charged.